[2021-07-16 10:48:11,299] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T10:46:57.896299+00:00 [queued]>
[2021-07-16 10:48:11,330] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T10:46:57.896299+00:00 [queued]>
[2021-07-16 10:48:11,331] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 10:48:11,332] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 10:48:11,333] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 10:48:11,370] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T10:46:57.896299+00:00
[2021-07-16 10:48:11,395] {standard_task_runner.py:52} INFO - Started process 2822 to run task
[2021-07-16 10:48:11,406] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T10:46:57.896299+00:00', '--job-id', '691', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpqa_wwlge', '--error-file', '/tmp/tmpvhygoi4f']
[2021-07-16 10:48:11,407] {standard_task_runner.py:77} INFO - Job 691: Subtask restart_test
[2021-07-16 10:48:11,511] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T10:46:57.896299+00:00 [running]> on host a2cbcfd4d6f4
[2021-07-16 10:48:11,620] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T10:46:57.896299+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T10:46:53.866465+00:00
[2021-07-16 10:48:14,613] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T104657, start_date=20210716T104811, end_date=20210716T104814
[2021-07-16 10:48:14,669] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 10:48:14,712] {local_task_job.py:151} INFO - Task exited with return code 0
